src/main.rs

```diff
@@ -130,11 +130,15 @@ fn main() -> Result<()> {
         let verb = Trust_subcommands::from_str(v);
         let app_id = arguments::get_app_id(&command.unwrap(), &context)?;
         let days = command.unwrap().value_of(&Parameters::days);
+        let key_input = match command.unwrap().value_of(&Parameters::key_input) {
+            Some(f) => Some(trust::verify_input_key(f)?),
+            _ => None,
+        };
```

Comment:
you can `map` the Option instead of matching only on Some
https://doc.rust-lang.org/std/option/enum.Option.html#method.map